src/app/type/type.component.html

```diff
@@ -0,0 +1,33 @@
+<!--Type Start-->
+<div class="type-panel-header pointer display-flex">
+  <h3 class="flex-1">Types</h3>
```

Comment:
Don't think `class="flex-1"` is necessary, cause all it does is sets `flex: 1` which is always the default anyways.